package.json

```diff
@@ -21,7 +21,7 @@
     "test:int": "jest -c ./test/jest.integration.config.ts",
     "test:e2e": "jest -c ./test/jest.e2e.config.ts",
     "test:sec": "jest -c ./test/jest.security.config.ts",
-    "test:mig": "jest -c ./test/jest.migration.config.ts"
+    "test:mig": "yarn build && jest -c ./test/jest.migration.config.ts"
   },
```

Comment:
Why build? You can use yarn dev for the migration commands also

Reply:
Since we call yarn dev like 30 times, it simply runs faster if we build the project first.

Reply:
Makes sense from a CI point of view, but if you're in the process of writing a migration it's annoying to have to build the project to run the test

Reply:
Sometimes I will do a yarn test to check if everything is still working correctly. I test:mig builds the project it's going to be way longer

Reply:
It's faster overall, that's the reason why I made this change. I'll revert it since I don't really care.